packages/ember-data/lib/system/model/model.js

```diff
@@ -1054,13 +656,11 @@ var Model = Ember.Object.extend(Ember.Evented, {
     successfully or rejected if the adapter returns with an error.
   */
   save: function() {
-    var promiseLabel = "DS: Model#save " + this;
-    var resolver = Ember.RSVP.defer(promiseLabel);
-
-    this.store.scheduleSave(this, resolver);
-
+    var model = this;
     return PromiseObject.create({
-      promise: resolver.promise
+      promise: this._internalModel.save().then(function() {
```

Comment:
seems like save fulfilling with model is a good default, any reason why internalModel.save does not?